src/app/_common-components/not-found/not-found.component.scss

```diff
@@ -1 +1,52 @@
-/* stylelint-disable */
+@use '/src/variables.scss' as *;
+@use '@angular/material' as mat;
+
+:host {
+    display: flex;
+    min-height: 100%;
+    background-image: url('/assets/img/background-about.png');
```

Comment:
make a pull from development please ; the background image has been change, the way we do it ; look at the about component